apps/api/package.json

```diff
@@ -34,8 +34,10 @@
     "@nestjs/swagger": "^7.3.0",
     "@nestjs/throttler": "^6.2.1",
     "@nestjs/websockets": "^10.3.7",
+    "@slack/bolt": "^3.13.1",
     "@socket.io/redis-adapter": "^8.3.0",
     "@supabase/supabase-js": "^2.39.6",
+    "api": "file:",
```

Comment:
Whats this?

Reply:
Sorry , I did not intend to add all of the package.json changes in the final commit. It has no significance whatsoever I was just testing/exploring.

Reply:
Okay, but you should remove it in your next commit :)